Doc/library/contextlib.rst

```diff
@@ -236,10 +236,10 @@ Functions and classes provided:
 
    For example, the output of :func:`help` normally is sent to *sys.stdout*.
    You can capture that output in a string by redirecting the output to an
-   :class:`io.StringIO` object::
+   :class:`io.StringIO` object. For convenience, the new stream is available
+   as the context variable::
```

Comment:
Is "context variable" the right term for this?

Reply:
That's a confusing term because Python has a [contextvars](https://docs.python.org/3/library/contextvars.html) module, which is about something else. I'm not sure there's an established term for the return value of `__enter__`; https://docs.python.org/3/reference/compound_stmts.html#with just talks about the "target".

Reply:
67c22ce24e re-words this. It's a touch verbose now, but definitely clearer.